LostFilmMonitoring.Web/script.js

```
@@ -1,331 +1,616 @@
-        } else {
-            document.getElementById("savingSpan").style.display = 'none';
-            document.getElementById("savedSpan").style.display = 'inline-block';
-            setTimeout(() => {
-                document.getElementById("savedSpan").style.display = 'none';
-              }, 1000);
+/**
+ * Escapes special characters in a string
+ * @param {string} str - The string to escape
+ * @returns {string} The escaped string
+ */
+const escape = (str) => {
```

Biome (1.9.4)

[error] Do not shadow the global "escape" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)